README_new.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+At babylon, we firmly believe that **transparency** and **flexibility** are core values that should be present in everything we do, and hiring the best person should reflect that belief.
+
+That's why we in the iOS team have open-sourced our recruitment process, for a fairer assessment process and so that candidates know exactly what to expect, all the way to the [questions we may ask during a face-to-face interview](TODO).
+
+We also aim to be as flexible and inclusive as possible, equally accepting applications from candidates who want to work on-site at our office in London, or who want to work remotely, or who have a limited amount of time to write a demo project, or who have open-source contributions.
```

Comment:
This is mixing to ideas, aka flexibility and inclusivity of work and of applying.

```We also aim to be as flexible and inclusive as possible, equally accepting applications from candidates who want to work on-site at our office in London or who want to work remotely. This is also reflected on the application submission, given that we accept both demo projects (new or old) and open-source contributions.```

Reply:
Respectfully, I like the original version better than your suggestion though I think the original version might be improved by removing "and inclusive as possible", as follows:

> We also aim to be as flexible by equally accepting applications from candidates who want to work on-site at our office in London, or who want to work remotely, or who have a limited amount of time to write a demo project, or who have open-source contributions.

Is that better?